ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-api-gnmi.j2

```diff
@@ -36,6 +36,33 @@
 {%             endfor %}
 {%         endif %}
 {%     endif %}
+
+{%     if management_api_gnmi.transport is arista.avd.defined %}
+
+| Transport | Destination | Port | gNMI SSL Profile | Tunnel SSL Profile | VRF | Local Interface | Local Port | Target ID |
+| --------- | ----------- | ---- | ---------------- | ------------------ | --- | --------------- | ---------- | --------- |
+{%         if management_api_gnmi.transport.grpc_tunnel is arista.avd.defined %}
+{%             for transport in management_api_gnmi.transport.grpc_tunnel %}
+{%                 if transport.name is arista.avd.defined %}
```

Comment:
same comments as for the eos template regarding:
* removing the check for name (as it becomes primary_key with the schema modification I suggested)
* the target cli can mix and match serial-number and id